packages/dashboard-core-plugins/src/linker/LinkerOverlayContent.scss

```diff
@@ -14,17 +14,38 @@
     $toast-outer-padding: $spacer-3;
 
     position: absolute;
-    right: $toast-outer-padding;
-    bottom: $toast-outer-padding;
     background: $modal-content-bg;
     max-width: 400px;
     color: $background;
     pointer-events: auto;
     box-shadow: $tooltip-box-shadow;
     z-index: $zindex-popover;
 
+    .btn-drag-handle {
+      cursor: grab;
+      border-radius: 0;
+      border: none;
+      position: absolute;
+      background: $gray-300;
+      color: $gray-600;
+      top: 50%;
+      left: 0;
+      height: 100%;
+      min-width: 1.5rem;
+      max-width: 2rem;
+      transform: translateY(-50%);
```

Comment:
As @dsmmcken mentioned, use flexbox here to position/center the button - you shouldn't need to use this transform (or many of these other properties).

`min-width` and `max-width` should be the same as well, and should be referencing a variable. Could probably just use `$spacer-4`.